ci/docker/Dockerfile

```
@@ -5,6 +5,7 @@ FROM docker-private.packages.nuxeo.com/nuxeo/nuxeo:${NUXEO_VERSION}
 ARG BUILD_TAG=unknown
 ARG SCM_REF=unknown
 ARG VERSION=unknown
+ARG NUXEO_WEB_UI_VERSION=unknown
```

This makes the image unbuildable if we don't give the version, I would be in favor of:

```suggestion
ARG NUXEO_WEB_UI_MP=nuxeo-web-ui
```